internal/deviations/deviations.go

```diff
@@ -740,3 +740,43 @@ func EnableFlowctrlFlag(dut *ondatra.DUTDevice) bool {
 func Ipv6RouterAdvertisementConfigUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetIpv6RouterAdvertisementConfigUnsupported()
 }
+
```

Comment:
Our goal is to eventually get rid of all deviation flags. Is there other way to avoid adding so many deviation flags?

Reply:
Deviation list has been trimmed to 5 deviations